package.json

```diff
@@ -18,12 +18,12 @@
 	"types": "./dist/main.d.ts",
 	"install-peers": "install-peers",
 	"scripts": {
-		"start": "npm-run-all --parallel watch app",
+		"start": "yarn build-fandom-player && npm-run-all --parallel watch app",
 		"build": "yarn build-fandom-player && yarn build-stand-alone",
 		"build-fandom-player": "yarn run create-dist && rollup -c",
 		"build-stand-alone": "cd stand-alone && yarn && yarn run create-standalone-dist && rollup -c rollup-stand-alone.config.js",
 		"deploy-stand-alone": "ts-node --project stand-alone/tsconfig-stand-alone.json stand-alone/deploy/deploy.ts",
-		"watch": "yarn run create-dist && NODE_ENV=development rollup -c --watch",
+		"watch": "NODE_ENV=development rollup -c --watch",
```

Comment:
do not remove the directory on watch to make the test app work immediately after starting